src/reporters/base.ts

```diff
@@ -200,11 +200,13 @@ function formatError(error: TestError, file?: string) {
       tokens.push(codeFrameColumns(source, {
         start: position,
       },
-      { highlightCode: true}
+      {
+        highlightCode: process.env.FORCE_COLOR === '1' || process.env.FORCE_COLOR === undefined,
```

Comment:
Why this logic? Perhaps you want `require('colors').enabled` instead?

Reply:
The reason was to have cleaner asserts in our tests. `require('colors').enabled` is better :+1:, didn't know about that.